mat64/inner.go

```diff
@@ -27,6 +27,17 @@ func Inner(x []float64, A Matrix, y []float64) float64 {
 	var sum float64
 
 	switch b := A.(type) {
+	case RawSymmmetricer:
```

Comment:
There seems be an extra 'm' in the interface name.

Also, we have an interface named Symmetric, which seems inconsistent with RawSymmetricer. Interface names are usually `<verb>-er` or `<noun>`, so I would favor RawSymmetric.